src/ViewModel/Converter/ArticleDownloadLinksListConverter.php

```diff
@@ -28,26 +29,45 @@ public function convert($object, string $viewModel = null, array $context = [])
     {
         $groups = [];
 
+        $downloads = [];
+        $types = [];
         if ($object->getPdf()) {
-            $items = [
-                new ViewModel\ArticleDownloadLink(new ViewModel\Link(
-                    'Article PDF',
-                    $this->downloadLinkUriGenerator->generate(DownloadLink::fromUri($object->getPdf())),
-                    false,
-                    ['article-identifier' => $object->getDoi(), 'download-type' => 'pdf-article']
-                )),
-            ];
+            $types[] = 'PDF';
+            $downloads[] = new ViewModel\ArticleDownloadLink(new ViewModel\Link(
+                'Article PDF',
+                $this->downloadLinkUriGenerator->generate(DownloadLink::fromUri($object->getPdf())),
+                false,
+                ['article-identifier' => $object->getDoi(), 'download-type' => 'pdf-article']
+            ));
 
             if ($object instanceof ArticleVor && $object->getFiguresPdf()) {
-                $items[] = new ViewModel\ArticleDownloadLink(new ViewModel\Link(
+                $downloads[] = new ViewModel\ArticleDownloadLink(new ViewModel\Link(
                     'Figures PDF',
                     $this->downloadLinkUriGenerator->generate(DownloadLink::fromUri($object->getFiguresPdf())),
                     false,
                     ['article-identifier' => $object->getDoi(), 'download-type' => 'pdf-figures']
                 ));
             }
+        }
+
+        if (!empty($context['dar-download'])) {
+            $types[] = 'DAR';
+            $downloads[] = new ViewModel\ArticleDownloadLink(
+                new ViewModel\Link(
+                    'Executable DAR',
+                    $this->downloadLinkUriGenerator->generate(DownloadLink::fromUri($context['dar-download'])),
+                    false,
+                    ['article-identifier' => $object->getDoi(), 'download-type' => 'dar-download']
+                ),
+                new ViewModel\Link(
+                    'Learn more about DAR',
+                    '#'
```

Comment:
empty placeholder? Guess fine for now to demonstrate the UI

Reply:
Meant to remove.

Reply:
https://github.com/elifesciences/issues/issues/5346#issuecomment-582901367